multiprofit/objects.py

```diff
@@ -729,16 +731,22 @@ def _getexposuremodelsetup(self, exposure, engine=None, engineopts=None, clock=F
                         if covarsrc is not None:
                             if clock:
                                 timeeig = time.time()
-                            reff, axrat, ang = mpfgauss.covartoellipse(
-                                covarpsf[0, 0] + covarsrc[0, 0],
-                                covarpsf[1, 1] + covarsrc[1, 1],
-                                covarpsf[0, 1] + covarsrc[0, 1],
-                            )
+                            covar = (covarpsf[0, 0] + covarsrc[0, 0],
+                                     covarpsf[1, 1] + covarsrc[1, 1],
+                                     covarpsf[0, 1] + covarsrc[0, 1])
+                            reff, axrat, ang = mpfgauss.covartoellipse(covar[0], covar[1], covar[2])
```

Comment:
I'll try to add a note earlier where I also had this issue, do you need the covariance as a 2x2 matrix?  Is this the default way it's returned elsewhere, and that's why it's used this way? It seems like you only ever use the [0, 1] off diagonal, and never use the matrix as a matrix.

Reply:
I changed this in DM-19843, but since you mentioned it, I think there's only one part of the code where it actually needs the matrix (using the eigen method to convert to r, axrat, ang), so I'm better off consistently keeping only three terms. Annoyingly, though, in some cases like that I need sigma_x^2, sigma_y^2, offdiag, whereas in others I need sigma_x, sigma_y, rho. I was thinking of consistently naming the former Terms (as in terms of the covariance matrix) and the latter Params (since they are the parameters I actually care about) but I am open to better suggestions since there are many other kinds of terms and parameters in use elsewhere.

Reply:
"variance-covariance vector" vs "correlation vector"?  Neither of those are particularly great, but they do describe things more than terms/params.